File: Practica_Material_Design/app/src/main/java/com/example/practica_material_design/MainActivity.java
package com.example.practica_material_design;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    private Button btnAddEmployee, btnLstEmployees;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        btnAddEmployee  = findViewById(R.id.btnAddEmployee);
        btnLstEmployees = findViewById(R.id.btnLstEmployees);


        btnAddEmployee.setOnClickListener(v -> {
                startActivity(new Intent(MainActivity.this, AddEmployeeActivity.class));
        });

        btnLstEmployees.setOnClickListener( v -> {
            startActivity(new Intent(MainActivity.this, ListEmployeesActivity.class));
        });
    }
}

File: CalculadoraKotlin/app/src/main/java/com/example/calculadorakotlin/MainActivity.kt
package com.example.calculadorakotlin

import android.content.Intent
import android.content.pm.ActivityInfo
import android.os.Bundle
import android.view.View
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity(), View.OnClickListener {

    private lateinit var btnCalc: Button
    private lateinit var btnDatosAlumno: Button
    private lateinit var btnSalir: Button
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        this.requestedOrientation =
            ActivityInfo.SCREEN_ORIENTATION_PORTRAIT /// Unicamente Modo Vertical


        btnCalc = findViewById(R.id.btnCalc)
        btnDatosAlumno = findViewById(R.id.btnDatosAlumno)
        btnSalir = findViewById(R.id.btnSalir)

        btnCalc.setOnClickListener(this)
        btnDatosAlumno.setOnClickListener(this)
        btnSalir.setOnClickListener(this)
    }

    override fun onClick(p0: View?) {
        when(p0!!.id){
            R.id.btnCalc -> {
                // Ir a Calculadora Activity
                startActivity(Intent(this.applicationContext, CalculadoraActivity::class.java))
            }
            R.id.btnDatosAlumno -> {
                // Ir a Datos del Alumno
                startActivity(Intent(this.applicationContext, datos_alumno::class.java))
            }
            R.id.btnSalir -> {
                // Salir
                finish();
            }
        }
    }
}